EpiAware/src/EpiLatentModels/broadcast/helpers.jl

```diff
@@ -8,7 +8,8 @@ Constructs a `BroadcastLatentModel` appropriate for modelling the day of the wee
 - `BroadcastLatentModel`: The broadcast latent model.
 "
 function broadcast_dayofweek(model::AbstractTuringLatentModel)
-    return BroadcastLatentModel(model, 7, RepeatEach())
+    transformed_model = TransformLatentModel(model, x -> 7 * softmax(x))
```

Comment:
do we really want to hard code this vs provide it as the default link? I think there is an arg either way and given that might as well make it an arg?

Reply:
I guess its sort of a debate about what helper functions are for: one definite thing or a reduction of options to a few sensible ones?